<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSecretMessageCreateDialog</class>
 <widget class="QDialog" name="QtSecretMessageCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create secret message</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: rgb(0, 0, 0);  }
QPushButton { background-color: rgb(0, 0, 0);  }
QPushButton { font-size: 24px; }
QPushButton { color: rgb(96,96,96); }
QPushButton { font: &quot;Courier New&quot; }
QLabel { background-color: rgb(0, 0, 0);  }
QLabel { font-size: 24px; }
QLabel { color: rgb(96,96,96); }
QLabel { font: &quot;Courier New&quot; }
QWidget { background-color: rgb(0, 0, 0);  }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_load_original">
       <property name="text">
        <string>(X) Load original</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_load_secret">
       <property name="text">
        <string>(X) Load secret message</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="text">
        <string>(X) Save result</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_load_original">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="spacer_load_original_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_original">
         <property name="text">
          <string>No original loaded yet</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_load_original_todo">
         <property name="text">
          <string>Click on 'Load original' to do so</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_load_original_bottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_load_secret">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="spacer_load_secret_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>196</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_secret">
         <property name="text">
          <string>No secret loaded yet</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_load_secret_todo">
         <property name="text">
          <string>Click on 'Load secret message' to do so</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_load_secret_bottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>195</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_save">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="spacer_save_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>179</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_result">
         <property name="text">
          <string>The resulting image</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_save_bottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
